volatility3/framework/plugins/linux/psaux.py

```diff
@@ -0,0 +1,90 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+from typing import Optional
+
+from volatility3.framework import symbols, exceptions, renderers, interfaces
+from volatility3.framework.objects import utility
+from volatility3.plugins.linux import pslist
+
+class PsAux(pslist.PsList):
```

Comment:
Errr, this isn't ideal unless you're actually going to call pslist (which we kind of do by calling `self.list_tasks`).   To prevent breakages, please drop the inheritance and call list_tasks (which should be a classmethod for this very reason).  You'll need to write your own `get_requirements` method, but you'd have to anyway, because there should be a `VersionRequirement` or `PluginRequirement` (they're practically identical) to make sure pslist does what you expect it to do.


Not inheriting just prevents this breaking if pslist changes in the future.  Strictly there should be a requirement on a specific version of pslist, but if you're going that far, you might as well